Assets/Scripts/Character/ItemHolder.cs

```diff
@@ -5,31 +5,46 @@
 public class ItemHolder : MonoBehaviour {
   [SerializeField] private Transform handTransform;
   [SerializeField] private Item _heldItem;
+  [SerializeField] private Collider holderCollider; 
 
   public Item HeldItem { get => _heldItem; }
 
   private Transform oldParent = null;
 
+  private void ignoreCollisions(Item item, bool ignore) {
+    foreach (Collider collider in item.GetComponentsInChildren<Collider>()) {
```

Comment:
Not a high priority but caching the collection of colliders for just the most recent item used by this function might be worth it.